doc/cygnus-ngsi/installation_and_administration_guide/performance_tips.md

```diff
@@ -36,6 +37,22 @@ Nevertheless, as explained above, it is highly recommended to increase at least
 
 [Top](#top)
 
+###<a name="section1.2"></a>Retries
+Batches could not be persisted. This is someting may occur from time to time because the sink is temporarilly not available, or the communications are failing. In that case, Cygnus has implemented a retry mechanism.
+
+Regarding the retries of not persisted batches, a couple of parameters is used. On the one hand, a Time-To-Live (TTL) is used, specifing the number of retries Cygnus will do before definitely dropping the event (0 means no retries, -1 means infinite retries). On the other hand, a list of retry intervals can be configured. Such a list defines the first retry interval, then se second retry interval, and so on; if the TTL is greater than the length of the list, then the last retry interval is repeated as many times as necessary.
+
+By default, all the sinks have a configured batch ttl and retry intervals of 10 and 5000 miliseconds, respectively. These are the parameters all the sinks have for these purpose:
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1324/commits/5761edf332b6de1a9bbc4ae3285193574a368c85